main_test.go

```diff
@@ -16,34 +16,32 @@ func TestFindRoot(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-
-	expect := filepath.Join(wd, "_testdata", "rootfind")
-
-	got1, err := findProjectRoot(expect)
+	want := filepath.Join(wd, "_testdata", "rootfind")
```

Comment:
as long as we're changing things at this level...

this particular bit of spacing was intentional - the first expect/want var is reused across several of the successive checks. it was set off in spaces to visually indicate that it is independent of that first stanza.